features/gethtml.yml

```
@@ -1,6 +1,7 @@
 name: getHTML()
 description: The `getHTML()` method of `Element` and `ShadowRoot` objects serializes an element's DOM to an HTML string.
 spec: https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#html-serialization-methods
+group: dom
```

`getHTML()` is specific to HTML:

```suggestion
group: html
```

---

Can you elaborate on this one a bit? I thought of this as "fancy `innerHTML`," which is part of the basic `dom` feature. It seemed weird to me that they would be in separate groups.

---

I think we should keep `dom` here. The `html` group is typically used for HTML elements, not for APIs.